src/robot/running/librarykeywordrunner.py

```diff
@@ -119,9 +120,12 @@ def _dry_run(self, context, args):
             self._handler.resolve_arguments(args)
 
     def _executed_in_dry_run(self, handler):
+        keywords_to_execute_in_dry_run = ('BuiltIn.Import Library',
+                                          'BuiltIn.Set Library Search Order',
+                                          'BuiltIn.Set Tags,'
+                                          'BuiltIn.Remove Tags')
```

Comment:
Here seems to be a bug. This illustrates why all new features, like this one, need a test.

Extracting the list of keywords to a variable is a good idea. Perhaps it could just be named `keywords_to_execute`, though.